backend/apid/apid.go

```diff
@@ -78,27 +70,23 @@ func New(c Config, opts ...Option) (*APId, error) {
 	var tlsConfig *tls.Config
 	var err error
 	if c.TLS != nil {
+		// TODO(palourde): We should avoid using the loopback interface
+		c.TLS.InsecureSkipVerify = true
```

Comment:
Should we make a copy for specifically for the GraphQL service? Otherwise we might be allowing it on the `apid`.

Reply:
InsecureSkipVerify is only for clients.

Reply:
Good point, I applied `InsecureSkipVerify` to `tlsConfig` instead; I believe it only applies to client but I prefer to avoid any potential problem.